### DISCUSSION TITLE
How to make y-axis don't show decimal values

### DISCUSSION THREAD

**Discussion by Kotletka89** [patrykandpatrick/vico#729] [Questions] [answered]
my data will always contain integers, but it shows float numbers like 0.5 1.5 2.5

**Comment by Gowsky** [comment, marked as answer]
Hello! You can use `AxisItemPlacer.Vertical.step({ 1f })`. [See here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/axes) for more information.

**Reply by Kotletka89** [reply]
thanks!
